robotbase/src/main/java/frc/robot/commands/scoring/coral/CoralHome.java

```diff
@@ -2,15 +2,16 @@
 
 import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
 import frc.robot.Constants;
-import frc.robot.commands.elevator.ElevatorSetDistance;
 import frc.robot.commands.laterator.LateratorSetDistance;
+import frc.robot.commands.laterator.LateratorZero;
 
 public class CoralHome extends ParallelCommandGroup {
 
   public CoralHome() {
     super(
       new LateratorSetDistance(Constants.LATERATOR.SETPOINTS.HOME),
-      new ElevatorSetDistance(Constants.ELEVATOR.SETPOINTS.HOME)
+      new LateratorZero()
```

Comment:
This should happen after the LateratorSetDistance command. This is a parallel command group